src/vocgui/admins/training_set_admin.py

```diff
@@ -102,7 +104,11 @@ def get_form(self, request, obj=None, **kwargs):
                 .order_by("level")
             )
             form.base_fields["documents"].queryset = Document.objects.filter(
-                created_by__in=[group.name for group in request.user.groups.all()]
+                Q(created_by__in=request.user.groups.all())
```

Comment:
```suggestion
                Q(created_by__in=[group.name for group in request.user.groups.all()])
```
I'm sorry, I didn't realize that `created_by` is a char field in the document model :see_no_evil: